src/core/AtomDecomposition.hpp

```diff
@@ -92,7 +92,9 @@ class AtomDecomposition : public ParticleDecomposition {
     return id_to_cell(p.identity());
   }
 
+  Utils::Vector3d max_cutoff() const override;
```

Comment:
this needs some docstring

Reply:
are docstrings not inherited?

Reply:
ah you are right, i overlooked the inheritance